### PR TITLE
Adds an igniter to the medical borg

### PR DESCRIPTION
**Pull request by ZombiesVsPlants**
## About The Pull Request

The Medical borg now gets an ignited

## Why It's Good For The Game

As a normal medical borg, it makes heater beakers much faster and possible to do even if someone has griefed the chemical heaters or power has run out (also makes it much less aggravating to make ash and charcoal)

Adding an igniter for antagonistic play will open up lots of interesting strategies for the medical borg instead of just injecting everyone and getting beat by those in a skinsuit.
With a mobile source of heat, you could: 
Splash napalm or oil onto your target and ignite them
Splash thermite onto a wall in order to get through
Use Stabilizing Agent and black powder to sacrifice a beaker and make a placeable explosion

TLDR: encourages creative plays.

## Changelog
:cl:
add: Added an igniter to the medical borg
/:cl:

### CODE REVIEW

**Comment by Singularity69** [comment]
inb4 medical borgs could make meth bombs

**Comment by ZombiesVsPlants** [comment]
They already could

**Comment by jupyterkat** [comment]
at least put it on emagged borgs only, having  a literal flamethrower or wall melting thermite on a fucking medborg, think about it.

**Comment by ZombiesVsPlants** [comment]
I am tempted, but if the mediborg is emagged then it's better off using its hacked hypospray
I'd like Terra's or Elon's opinion before changing
Also how the heck do you make a flamethrower? that sounds cool as hell!

**Comment by TheFakeElon** [comment]
> TLDR: encourages creative plays.

Splashing napalm and thermite is far from creative 😆.

This is dumb, med borgs do not need it. contrary to the belief of a prevalent silicon main; they're for healing not validhunting and r-wall breaching.

Almost all medicines can be made without requiring heat, the ones that do almost always have an alternative, if someone had the foresight to deconstruct chem heaters for whatever reason then tough luck 😛

File: austation/code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -384,3 +384,32 @@
 		else
 			return FALSE
 	return ..()
+
+
+
```
Comment (TheFakeElon) [inline austation/code/modules/mob/living/silicon/robot/robot_modules.dm:389]:
```suggestion
```

**Comment by ZombiesVsPlants** [comment]
Eh, you can still add heat to mixes its just confusing and annoying, reopening

**Comment by MCterra10** [comment]
unneeded. all this is good for is combat shit. it has no viable usecase for a mediborg's actual job; healing
